modules/_labs/pagination/react/stories/stories_visualTesting.tsx

```diff
@@ -27,103 +27,94 @@ const TableRenderer = ({direction = ContentDirection.LTR}) => {
   const lastPage = getLastPage(resultCount, totalCount);
 
   return (
-    <CanvasProvider theme={{canvas: {direction}}}>
-      <StaticStates>
-        <ComponentStatesTable
-          rowProps={[
-            {label: 'Step Controls', props: {}},
-            {label: 'Jump Controls', props: {shouldShowJumpControls: true}},
-            {label: 'GoTo Form', props: {shouldShowJumpControls: true, shouldShowGoToForm: true}},
-            {
-              label: 'Additional Details',
-              props: {
-                shouldShowJumpControls: true,
-                shouldShowGoToForm: true,
-                shouldShowAddtionalDetails: true,
-              },
+    <StaticStates theme={{canvas: {direction}}} style={{display: 'inline-block'}}>
```

Comment:
This whole changeset is removing the `CanvasProvider` component and allowing `theme` to be forwarded through `StaticStates` instead. `StaticStates` and `CanvasProvider` also forward extra attributes to the direction tag used in the `CanvasProvider` component.